### PR TITLE
🐛Throw exception on S3 writing error

### PR DESCRIPTION
**Pull request by shepilov**

### CODE REVIEW

File: tdrive/frontend/src/app/features/auth/provider/oidc/oidc-auth-provider-service.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export default class OIDCAuthProviderService
         redirect_uri: getAsFrontUrl(OIDC_CALLBACK_URL),
         response_type: 'code',
         response_mode: 'query',
-        scope: 'openid profile email address phone offline_access',
+        scope: 'openid profile email address phone offline_access applications',
```
Comment (chibenwa) [inline tdrive/frontend/src/app/features/auth/provider/oidc/oidc-auth-provider-service.ts:56]: How is this related to the problem at hand?

**Review by chibenwa — COMMENTED** [review]
Where is the unit test?

In James we can reproduce outages in tests by pausing dockers, for instance.

File: tdrive/frontend/src/app/features/auth/provider/oidc/oidc-auth-provider-service.ts (hunk 1 above)
Comment (shepilov) [reply tdrive/frontend/src/app/features/auth/provider/oidc/oidc-auth-provider-service.ts:56]: @chibenwa  It's a quick fix, maybe also some comments about code style?

Comment (chibenwa) [reply tdrive/frontend/src/app/features/auth/provider/oidc/oidc-auth-provider-service.ts:56]: No I ask why we need to change the OIDC scope on an error about S3 handling.

This seems unrelated to me, I just wonna make sure it is not done by mistake.
